sqlmesh/core/engine_adapter/base.py

```diff
@@ -481,15 +489,82 @@ def _create_table_from_columns(
             if primary_key and self.SUPPORTS_INDEXES
             else []
         )
-        schema = exp.Schema(
-            this=table,
-            expressions=[
-                exp.ColumnDef(this=exp.to_identifier(column), kind=kind)
-                for column, kind in columns_to_types.items()
-            ]
-            + primary_key_expression,
+
+        column_descriptions = kwargs.pop("column_descriptions", None)
```

Comment:
We should probably make `column_descriptions` and `table_description` actual named arguments of the `_create_table_from_columns` method.